UI/js-src/lsmb/menus/Cache.js

```diff
@@ -0,0 +1,41 @@
+define(["dojo/_base/lang","dojo/when" /*=====, "dojo/_base/declare", "dojo/store/api/Store" =====*/],

```

Comment:
Why do we need  a specific cache when we already implement a very specific REST store below?